src/app/business/delfin/storages/storages.component.html

```diff
@@ -0,0 +1,528 @@
+                                        </span> 
+                                        <span>/</span>
+                                        <span class="used-storage">
+                                            {{device.capacity && device.capacity.system_used ? device.capacity.system_used : '-'}} 
+                                            
+                                        </span> 
+                                        <span>/</span>
+                                        <span class="total-storage">
+                                            {{device.capacity && device.capacity.raw ? device.capacity.raw : '-'}} 
+                                            
+                                        </span>
+                                    </div>
+                                </div>
+                                
+                            </div>
+                        </p-scrollPanel>
+                    </div>
+                    <p-footer>
+                        <hr />
+                        <span>Last updated: <strong> </strong></span>
+                    </p-footer>
+                </p-card>
+            </div>
+        </div>
+    </div>
+    <div *ngIf="showListView">
+        <div id="storage-devices-table" class="ui-g">
+            <p-card  styleClass="ui-card-shadow">
+                <p-dataTable [value]="allStorages" [globalFilter]="searchAll" [(selection)]="selectedStorages" [expandableRows]="true" [showHeaderCheckbox]="true" [rows]="10" [paginator]="true" [pageLinks]="3" [rowsPerPageOptions]="[10,20,50,100]" #st>
+                    <p-column expander="true" styleClass="col-icon" [style]="{'width': '3%'}"></p-column>
+                    <p-column selectionMode="multiple" [style]="{'width': '3%'}"></p-column>
+                    <p-column field="name" header="Name" [style]="{'width': '10%'}" [sortable]="true">
+                        <ng-template pTemplate="body" let-storage="rowData">
+                            <a [routerLink]="['/storageDetails/' + storage.id]"  title="{{storage.name}}">{{storage.name}}</a>
+                        </ng-template>
+                    </p-column>
+                    <p-column field="vendor" header="Vendor" [style]="{'width': '10%'}" [sortable]="true">
+                    </p-column>
+                    <p-column field="model" header="Model" [style]="{'width': '8%'}" [sortable]="true"></p-column>
+                    <p-column field="status" header="Status" [style]="{'width': '8%'}">
+                        <ng-template pTemplate="body" let-storage="rowData">
+                            <span class="storage-status-icon" [ngClass]="{normal: storage.status=='normal', abnormal: storage.status=='abnormal' || storage.status=='offline'}"><i class="fa fa-circle"></i></span>
+                            <span *ngIf="storage.status=='normal'">Normal</span>
+                            <span *ngIf="storage.status=='abnormal'">Abnormal</span>
```

Comment:
What is this status Abnormal? I don't think we should show the status as Abnormal.. It can be "Error"

Reply:
This is as per the [API spec](https://github.com/sodafoundation/delfin/blob/b0459fc0c44c13f4e4a8937a7caa22c747d6b559/openapi-spec/swagger.yaml#L85).

![image](https://user-images.githubusercontent.com/19162717/93019882-6fd24580-f5f7-11ea-8565-0911bdaf5d1b.png)

Reply:
Please check with @NajmudheenCT , but IMO you can simply do 
```
 <span *ngIf="storage.status=='Error'">Abnormal</span>
```

Reply:
Checking with other stake holders , Need a unified term for 'Error' , 'Failure' , 'Unknown' etc.. any status which is not normal is mapped to 'abnormal' now..

Reply:
Let me know the state machine and the values to display to the user . At the moment we have the following:
normal -> Normal
abnormal -> Abnormal
offline -> Offline

Reply:
IMHO, if !normal = Error, unless if we report the specifics

Reply:
@anvithks please open an issue to track reporting the status properly.

Reply:
@NajmudheenCT @kumarashit 
Raised #400 to track this.
I would suggest getting the status codes fixed at the earliest. There will be mulitple places where the status code will need to be checked and appropriate style may be applied or operation may be allowed / disallowed.